web_last_viewed_records/__openerp__.py

```diff
@@ -0,0 +1,15 @@
+{
+    'name': 'Last viewed records',
+    'version': '1.0.0',
+    'author': 'Ivan Yelizariev, Odoo Community Association (OCA)',
+    'category': 'Base',
+    'website': 'https://yelizariev.github.io',
+    'depends': ['web', 'mail'],
```

Comment:
There is a reference to mail at static/src/xml/main.xml
Mail inbox page use own template which has to be extended in a different way.

Option to exclude this dependencies is to make another module
**web_last_viewed_records_mail**
with a settings

    'depends': ['mail', 'web_last_viewed_records'],
    'auto_install': True

and that module would containt only this xml:


    <?xml version="1.0" encoding="UTF-8"?>
    <templates>
    	<t t-extend="mail.wall">
    		<t t-jquery="tr.oe_header_row_top" t-operation="after">
    			<t t-call="web_last_viewed_records.header">
    				<t t-set="colspan" t-value="2"/>
    			</t>
    		</t>
    	</t>
    </templates>